src/without-bang-important.js

```diff
@@ -0,0 +1,23 @@
+// Module with the same interface as the core aphrodite module,
+// except that styles injected do not automatically have !important
+// appended to them.
+//
+import {injectAndGetClassName} from './inject';
+
+import {
+    StyleSheet,
+    StyleSheetServer,
+    StyleSheetTestUtils,
+} from './index.js'
+
+const css = (...styleDefinitions) => {
+    // Do not append !important to style definitions
+    return injectAndGetClassName(false, styleDefinitions);
```

Comment:
In personal experience, comments only tend to get out of date when they're far removed from the thing they apply to spacially (e.g. a comment discussion caveats of function use). I've rarely seen comments a single line above what they apply to get out of date when you have a sane code reviewing process in place.

I do see your point about the non-obvious connection between the comment and the `false` here, so I'm happy to take the hybrid approach and include the variable too.